08-TabbedNavigation/src/PrismSample/Views/TabsPage.xaml

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<TabbedPage xmlns="http://xamarin.com/schemas/2014/forms"
+            xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+            xmlns:prism="http://prismlibrary.com"
+            xmlns:views="clr-namespace:PrismSample.Views;assembly=PrismSample"
+            prism:ViewModelLocator.AutowireViewModel="True"
```

Comment:
Do not include the ViewModelLocator.AutowireViewModel... it's not needed for any page in this sample!